packages/selection/src/utils/ACTIONS.js

```diff
@@ -0,0 +1,17 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+export default {
```

Comment:
Throughout the PR there's often use of default exporting objects. You've probably got a good reason for this, but it might be preferable to use named exports so you can get module object introspection so you get compile time errors instead of runtime errors on bad imports/access.

Reply:
Addressed in 0deb36a